doc/costFunctions.yaml

```diff
@@ -0,0 +1,458 @@
+    Lift stability pitch derivative
+    TODO: Someone who knows aircraft stability should verify these
+
+cfy0:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cfyalpha:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+    ??
+
+cfyalphddot:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cd0:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cdalpha:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cdalphadot:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+
+cmzq:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cmzqdot:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+clq:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+clqdot:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    TODO: Someone who knows aircraft stability should verify these
+
+cbend:
+  desc: >
+    NOTE: Time spectral stability derivatives are broken as of 2022.
+    Bending coefficient defined as :math:`\sigma L^2/ (qA)`
+    TODO: Someone who knows aircraft stability should verify these
+
+sepsensor:
+  desc: >
+    The separation values for the given surface is provided by this cost function.
+    Please see the below paper for further details.
+    See :cite:t:`Kenway2017b`.
+
+sepsensoravgx:
+  desc: >
+    The separation sensor average in x direction.
+    The sensor times the distance in x direction value is computed here.
+
+sepsensoravgy:
+  desc: >
+    The separation sensor average in y direction.
+    The sensor times the distance in y direction value is computed here.
+
+sepsensoravgz:
+  desc: >
+    The separation sensor average in z direction.
+    The sensor times the distance in z direction value is computed here.
+
+cavitation:
+  desc: >
+    Cavitation sensor, not to be confused with the cavitation number.
+    It is a modified Heaviside function accounting for how much :math:`-C_p` exceeds the cavitation number over the given surface family.
+    The `computeCavitation` flag must be set to `True` because this introduces additional computations.
+    You probably will specify the surface family with `cgnsutilities`.
```

Comment:
I think we should use double (``) around "computeCavitation" and "cgnsuitlities".  Currently they show up as italicized in the docs instead of inline code.